sct.py

```diff
@@ -581,8 +581,11 @@ def list_resources(ctx, user, test_id, get_all, get_all_running, verbose):
 def list_images(cloud_provider: str, branch: str, version: str, region: str, arch: AwsArchType):
     add_file_logger()
     branch_fields = ["Backend", "Name", "ImageId", "CreationDate", "BuildId", "Arch", "ScyllaVersion"]
+    branch_fields_with_name_tag = [
+        "Backend", "Name", "ImageId", "NameTag", "CreationDate", "BuildId", "Arch", "ScyllaVersion"]
     #  TODO: align branch and version fields once scylla-pkg#2995 is resolved
     version_fields = ["Backend", "Name", "ImageId", "CreationDate"]
+    version_fields_with_name_tag = ["Backend", "Name", "ImageId", "NameTag", "CreationDate"]
```

Comment:
I would expect something like following for the current case:
```
version_fields = ["Backend", "Name", "ImageId", "CreationDate"]
version_fields_with_name_tag = version_fields + ["NameTag"]
```

The idea is that we should avoid duplicating the literals definitions.
Using proposed variant we will change a tag name just in one place when we need to. 
Using the current approach from the PR it requires change of 2 places among longer strings. It is more error prone coding for future updates.

Moreover, definition of additional lists is not really needed. It may be used just in place:

```
version_fields = ["Backend", "Name", "ImageId", "CreationDate"]
...
# aws
create_pretty_table(rows=rows, field_names=version_fields + ["NameTag"]).get_string(...)
...
# gce
create_pretty_table(rows=rows, field_names=version_fields).get_string(...)
...
```

So, please, reuse literals.